model/annotations/task_annotations.go

```diff
@@ -26,17 +27,9 @@ type IssueLink struct {
 }
 
 type Source struct {
-	Author string    `bson:"author,omitempty" json:"author,omitempty"`
-	Time   time.Time `bson:"time,omitempty" json:"time,omitempty"`
-}
-
-type Annotation struct {
-	// comment about the failure
-	Note *Note `bson:"note,omitempty" json:"note,omitempty"`
-	// links to tickets definitely related.
-	Issues []IssueLink `bson:"issues,omitempty" json:"issues,omitempty"`
-	// links to tickets possibly related
-	SuspectedIssues []IssueLink `bson:"suspected_issues,omitempty" json:"suspected_issues,omitempty"`
+	Author    string    `bson:"author,omitempty" json:"author,omitempty"`
+	Time      time.Time `bson:"time,omitempty" json:"time,omitempty"`
+	Requester string    `bson:"requester,omitempty" json:"requester,omitempty"`
```

Comment:
We want to be able to do checks like `if source.requester == api`, how can we make it more precise than a string?

Reply:
I created constants for the requester types, so that itself will ensure consistency (this is how we handle patch requesters, for example). The reason I chose this approach rather than an enum is because that translates to just a number, which isn't very intuitive to read -- you'd have to look up in the docs if 1 corresponded to UI or API.

Reply:
We could create another type RequesterType and wrap string